src/modules/Kdump.rb

```diff
@@ -38,6 +38,10 @@ class KdumpClass < Module
     KDUMP_SERVICE_NAME = "kdump"
     KDUMP_PACKAGES = ["kexec-tools", "kdump"]
 
+    # Space on disk reserved for dump additionally to memory size
+    # @see FATE #317488
+    RESERVED_DISK_SPACE_BUFFER = 4 * 1024**3
```

Comment:
Please explicitly mention the units in the name: `_B`, `_MB`. Here I can guess it from the value, but we may get it wrong elsewhere.